From 3da9351e753fb0ac1489182b9b1ff1b037d8122d Mon Sep 17 00:00:00 2001
From: Richard Russon <rich@flatcap.org>
Date: Mon, 30 May 2016 03:42:06 +0100
Subject: [PATCH 02/13] prefix-mailcap

# User Fabian Groffen <grobian@gentoo.org>
# Date 1279820983 -7200

Adjust mailcap search path for Prefix (offset installs).  Due to the
search path addition, should not cause any problems for non-offset
installs.

---
 Makefile.am | 2 +-
 init.c      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index ba32e0b..b259f99 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -46,7 +46,7 @@ mutt_DEPENDENCIES = $(MUTT_LIB_OBJECTS) $(LIBOBJS) $(LIBIMAPDEPS) \
 
 DEFS=-DPKGDATADIR=\"$(pkgdatadir)\" -DSYSCONFDIR=\"$(sysconfdir)\" \
 	-DBINDIR=\"$(bindir)\" -DMUTTLOCALEDIR=\"$(datadir)/locale\" \
-	-DHAVE_CONFIG_H=1
+	-DHAVE_CONFIG_H=1 -DPREFIX=\"$(prefix)/..\"
 
 AM_CPPFLAGS=-I. -I$(top_srcdir) $(IMAP_INCLUDES) $(GPGME_CFLAGS) -Iintl
 
diff --git a/init.c b/init.c
index c9a6dd2..7ed4f7b 100644
--- a/init.c
+++ b/init.c
@@ -3327,7 +3327,7 @@ void mutt_init (int skip_sys_rc, LIST *commands)
   else
   {
     /* Default search path from RFC1524 */
-    MailcapPath = safe_strdup ("~/.mailcap:" PKGDATADIR "/mailcap:" SYSCONFDIR "/mailcap:/etc/mailcap:/usr/etc/mailcap:/usr/local/etc/mailcap");
+    MailcapPath = safe_strdup ("~/.mailcap:" PKGDATADIR "/mailcap:" SYSCONFDIR "/mailcap:" PREFIX "/etc/mailcap:/etc/mailcap:/usr/etc/mailcap:/usr/local/etc/mailcap");
   }
 
   Tempdir = safe_strdup ((p = getenv ("TMPDIR")) ? p : "/tmp");
-- 
2.8.2

